src/org/labkey/test/tests/AbstractExportTest.java

```diff
@@ -51,17 +51,35 @@ public abstract class AbstractExportTest extends BaseWebDriverTest
     protected DataRegionTable dataRegion;
     protected DataRegionExportHelper exportHelper;
 
-    /** Return true if the rows can be selected in the grid. */
+    /**
+     * Return true if the rows can be selected in the grid.
+     */
     protected abstract boolean hasSelectors();
+
+    protected abstract boolean hasBrokenLookup();
+
```

Comment:
This seems redundant. Checking whether `getTestLookUpColumnHeader` is `null` would work.

Reply:
good point, deleting the method